rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,194 @@
+# RFC 050: Design considerations for the concepts API
+
+This RFC collects some initial thinking on how we might represent concepts in the catalogue API.
+It's a starting point for discussions; not a final design.
+
+
+
+## Goals
+
+*   We want to be able to create per-person and per-subject pages on the website.
+    The data on these pages should come from the new concepts API.
+
+*   These pages should contain enough context to distinguish two people/subjects (e.g. whether "Charles Darwin" is the famous naturalist, his uncle, his son, or his grandson).
+
+*   We're not trying to replicate Wikidata -- the APIs shouldn't return large amounts of information we aren't going to use on wellcomecollection.org.
+
+*   Any new APIs should be similar to the existing works and images APIs.
+    We'll use the same conventions for filters, aggregations, including optional fields, and so on.
+
+
+
+## Endpoint #1: retrieve a single concept
+
+This is the single concept endpoint, which is used to render a page for an individual subject or person:
+
+```
+GET /concepts/azxzhnuh
+{
+  "id": "azxzhnuh",
+  "identifiers": [
+    {
+      "identifierType": "lc-names",
+      "value": "n12345678",
+      "type": "Identifier"
+    },
+    ...
+  ],
+
+  "label": "Florence Nightingale",
+  "alternativeLabels": [
+    "The Lady with the Lamp",
+  ]
+
+  "type": "Person|Subject|Organisation|Place"
+
+  // Everything above here is stuff I'm pretty sure we'll need;
+  // everything below it is more nebulous and more likely to change
+  // in the final API.
+
+  "description": "Florence Nightingale was an English social reformer, statistician and the founder of modern nursing. Nightingale came to prominence while serving as a manager and trainer of nurses during the Crimean War, in which she organised care for wounded soldiers at Constantinople.",
+
+  // not locations
+  "urls": [
+    "https://en.wikipedia.org/wiki/Florence_Nightingale"
+  ],
+
+  // cf productionEvent?
+  "dates": [ { date, meaning } ]
+  "places": [ { place, meaning } ]
+  "birthDate/place"
+  "deathDate/place"
+
+  "thumbnail": { … },
+
+  "relatedConcepts": [
+    {
+      "id": "asoiham1",
+      "label": "Crimea",
+      "type": "Place"
+    }
+  ]
```

Comment:
Can we take this out of scope for now? Talking about representing this in the API (let alone how to architect it internally) could occupy many hectares of bikesheds

Reply:
I've clarified it slightly – I've used a deliberately worse name (`connectedConcepts`), because I agree we don't want to be deciding the exact representation yet – but I do think we should decide whether you can get connections through this endpoint, or if they're available from a separate endpoint.

I've added something to the explanation of this field:

> Q: Do we want the connections between concepts to be discoverable from a single concept endpoint, or do we want that elsewhere?
> I don't think we need to decide exactly how we'll model it (yet), but we should decide if it lives here or in at a distinct endpoint.

This feels like something we should decide as part of the list of the new endpoints.